frontend/src/scenes/settings/user/PersonalAPIKeys.tsx

```diff
@@ -339,6 +339,12 @@ function PersonalAPIKeysTable(): JSX.Element {
                         )
                     },
                 },
+                {
+                    title: 'Key',
+                    dataIndex: 'mask_value',
+                    key: 'mask_value',
+                    render: (_, key) => key.mask_value,
```

Comment:
All existing keys will have a null `mask_value`, so we need to elegantly support that case too

Reply:
I have added "Preview unavailable" when the `mask_value` is not set in database